src/Components/ProductListing/CartPageListing.jsx

```diff
@@ -0,0 +1,32 @@
+import "../../styles/ProductListing.css";
+import React from "react";
+import {Cartitemcard} from "../Cards/Cartitemcard";
+import {useEffect ,useState} from "react";
+import axios from "axios";
+;
+
+const ProductListing = () => {
+  const [products, setProducts] = useState([]);
+  async function getItems() {
+    try {
+      const res = await axios.get("/api/products");
+      console.log(res.data);
+      setProducts(res.data.products);
+    } catch (error) {
+      console.log(error);
+    }
+  }
```

Comment:
Try abstracting this function out in a different folder (Items-utility or api-utility)